x-pack/plugins/security_solution/public/detections/hooks/trigger_actions_alert_table/use_actions_column.tsx

```diff
@@ -109,6 +109,6 @@ export const getUseActionColumnHook =
 
     return {
       renderCustomActionsRow,
-      width: 124,
+      width: 150,
```

Comment:
this value should be calculated based on ACTION_BUTTON_COUNT, not static. ACTION_BUTTON_COUNT * 31 i think?